dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/JobProgress.java

```diff
@@ -306,24 +306,42 @@ default boolean runStage( Map<String, Runnable> items )
         return true;
     }
 
+    /**
+     * @see #runStage(Function, Runnable)
+     */
+    default boolean runStage( Runnable work )
+    {
+        return runStage( null, work );
+    }
+
     /**
      * Run a stage with no individual work items but a single work
      * {@link Runnable} with proper completion wrapping.
      * <p>
      * If the work task throws an {@link Exception} the stage is considered
      * failed otherwise it is considered complete when done.
      *
+     * @param summary used when stage completes successful (return value to
```

Comment:
`successfully` AFAIK

Reply:
Yes, I'll try to remember it and get it in one of the upcoming PRs around jobs.